### ISSUE TITLE
Rutracker [Error] We not authorized, please check your credentials!

### ISSUE THREAD

**Issue by mysteryjet51** [imDMG/qBt_SE#27]
Kinozal works fine for me with credentials, however rutracker not. Everyone has the same problem?

**Comment by imDMG** [comment]
Try this https://github.com/imDMG/qBt_SE/issues/22

**Comment by mysteryjet51** [comment]
Спасибо, сработало после того как отключил rutracker прокси в браузере и несколько раз перезашёл в личном кабинете, юзая VPN установленный на компе.

**Comment by Foxtrod89** [comment]
It's not safe to store credentials in text files. Can we pull data from macOS's keychain instead?

**Comment by imDMG** [comment]
Yep, but you always need to keep the jackett enabled in the system tray.

**Comment by enigmachyrsalis** [comment]
I am seeing this same error despite entering my credentials in the .json file and logging out of RuTracker and restarting qBittorent.
I masked my UN and PW, but this is the code. Is this correct or is there a fix?

{
    "username": "********",
    "password": "********",
    "torrentDate": true,
    "proxy": false,
    "proxies": {
        "http": "",
        "https": ""
    },
    "ua": "Mozilla/5.0 (X11; Linux i686; rv:38.0) Gecko/20100101 Firefox/38.0 "
}

**Comment by imDMG** [comment]
You need to paste in json file your credentials from RuTracker (login/password).

**Comment by enigmachyrsalis** [comment]
> You need to paste in json file your credentials from RuTracker (login/password).

I didn't realize my post deleted the * character I had used. I edited my post to show the UN & PW as X's instead.
